benches/bench.rs

```diff
@@ -59,6 +63,14 @@ impl<T: Copy> Vector<T> for SmallVec<[T; VEC_SIZE]> {
         self.push(val)
     }
 
+    #[cfg(feature = "push_light")]
+    fn push_light(&mut self, val: T) {
+        self.push_light(val)
+    }
+
+    #[cfg(not(feature = "push_light"))]
+    fn push_light(&mut self, _: T) {}
```

Comment:
I don't understand why this is defined if the feature is missing, and why it does nothing.

Reply:
It's to ensure things still compile

Reply:
It compiles sure, but it doesn't do the right thing, why should it be a noop if the feature is missing? Sounds like a way to have silent bugs.

Reply:
It's not meaningful to benchmark a feature if it's disabled

Reply:
Oh ok sorry I didn't notice this was an implementation for benchmarks.